app/helpers/mobile_helper.rb

```diff
@@ -26,7 +26,7 @@ def mobile_like_icon(post)
   end
 
   def mobile_comment_icon(post)
-    link_to '', new_post_comment_path(post), :class => "image_link comment_action inactive"
+    link_to '', post_comments_path(post, :format => "mobile"), :class => "image_link comment_action inactive"
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.